baseplate/sidecars/event_publisher.py

```diff
@@ -86,10 +86,21 @@ def reset(self) -> None:
         self._size = len(self._header) + len(self._end)
 
 
+class V2JBatch(V2Batch):
```

Comment:
could you add a test here please? https://github.com/reddit/baseplate.py/blob/develop/tests/unit/lib/events/publisher_tests.py#L40

just something simple to exercise this code and make sure it doesn't regress in the future